lib/vsc/mympirun/rm/sched.py

```diff
@@ -49,6 +49,14 @@ def what_sched(requested):
                 return sched, found_sched
         LOGGER.warn("%s scheduler was requested, but mympirun failed to find an implementation", requested)
 
+    # if PBS_NODEFILE is not present, use local scheduler
+    filename = os.environ.get('PBS_NODEFILE', None)
+    if filename is None:
+        for sched in found_sched:
+            if sched._is_sched_for("local"):
+                LOGGER.debug("PBS_NODEFILE not found, using local scheduler")
+                return sched, found_sched
```

Comment:
this same loop is also used below, please don't duplicate code but factor it out in a function you call out to